app/metrics.yaml

```diff
@@ -1625,67 +1610,6 @@ activation:
     no_lint:
       - USER_LIFETIME_EXPIRATION
 
-qr_scanner:
```

Comment:
cc @eliserichards for qr scanner telemetry - when chatting with vesta we had the question about whether we ported the original QR telemetry over to the new search experience. If it's not relevant to the new experience then we wanted to remove it.

Reply:
ping @eliserichards